source/common/config/subscription_factory_impl.cc

```diff
@@ -130,13 +130,15 @@ SubscriptionPtr SubscriptionFactoryImpl::collectionSubscriptionFromUrl(
           Utility::configSourceInitialFetchTimeout(config), false);
     }
     default:
+      // TODO(htuch): Add support for non-aggregated delta gRPC, but there will always be options,
+      // e.g. v2 REST, that don't make sense for xdstp:// ResourceLocators.
```

Comment:
Are we going to support non-delta with XDSTP?

Reply:
There's no reason this can't be done, but it's not currently an implementation priority.